scipy/stats/_morestats.py

```diff
@@ -4467,39 +4454,22 @@ def circmean(samples, high=2*pi, low=0, axis=None, nan_policy='propagate'):
     >>> plt.show()
 
     """
-    samples, sin_samp, cos_samp, nmask = _circfuncs_common(samples, high, low,
-                                                           nan_policy=nan_policy)
-    sin_sum = sin_samp.sum(axis=axis)
-    cos_sum = cos_samp.sum(axis=axis)
+    samples, sin_samp, cos_samp = _circfuncs_common(samples, high, low)
+    sin_sum = sin_samp.sum(axis)
+    cos_sum = cos_samp.sum(axis)
```

Comment:
In the future let's keep passing `axis` as a keyword argument. That's how it is in the array API... well and I suppose it's also required to be used as a function there. No need to change here, though.